content/docs/monitoring/monitor_shared_datasets.md

```diff
@@ -37,15 +37,15 @@ The two extraction tasks are [scheduled]({{@link schedules.md}}) to run every da
 
 ![Joke storage]({{@asset monitoring/images/joke-storage.webp}})
 
-Now, to the monitoring part. For this tutorial, let's skip the monitoring of the tasks and jump right to the the dataset.
+Now, to the monitoring part. For this tutorial, let's skip the monitoring of the tasks and jump right to the dataset.
 
 ## [](#create-a-new-monitoring-task) Create a new monitoring task
 
 If you haven't already, [add the monitoring suite to your account]({{@link monitoring.md#add-the-monitoring-suite-to-your-account}}).
 
 If you have already added the task, under its **Settings** tab, give it a name. For example, **monitoring-jokes**.
 
-> We recommend prefixing your monitoring task names with **monitoring-** so you could identify them easier.
+> We recommend prefixing your monitoring task names with `monitoring-` so you could identify them easier.
```

Comment:
We reserve the code marks just for code, so for text, I'd prefer to use the double asterisk. Unless this name is in code, in which case we have a dilemma :D 
```suggestion
> We recommend prefixing your monitoring task names with **monitoring-** so you could identify them easier.
```